README.md

```diff
@@ -1,80 +1,6 @@
 # React Native in the Community
 
-A list of community-built React Native contributions. You can subscribe to updates on the highlights of the changes to this repo week-by-week, along with other news from the community in the [react-native-newsletter](https://github.com/brentvatne/react-native-newsletter).
-
-For a searchable list of npm-registered components, please see [react.parts](http://react.parts/native-ios)
-
-## Components
-
-> WARNING: This list is deprecated. The definitive list can be found at [react.parts](http://react.parts/native-ios)
-
-### Audio / Visual
-
-- Camera: https://github.com/lwansbrough/react-native-camera
-- Video: https://github.com/brentvatne/react-native-video
-- Audio Recording: https://github.com/jsierles/react-native-audiorecorder
-
-### Controls / Forms
-
-- Form handling: https://github.com/gcanti/tcomb-form-native
-- UISegmentedControl: http://qiita.com/laiso/items/5c5ef9158a37726d109d
-- CalendarPicker: https://github.com/stephy/CalendarPicker
-
-### Graphics
-
-- BlurView: https://github.com/Kureev/react-native-blur
-- Icons: https://github.com/corymsmith/react-native-icons
-- LinearGradient: https://github.com/brentvatne/react-native-linear-gradient
-- SVG (Experimental!): https://github.com/brentvatne/react-native-svg
-
-### Lists
-
-- Refreshable ListView: https://github.com/jsdf/react-native-refreshable-listview
-- Selectable Sections Listview: https://github.com/johanneslumpe/react-native-selectablesectionlistview
-
-### Navigation
-
-- Navbar: https://github.com/Kureev/react-native-navbar
-- Sidebar: https://github.com/dzannotti/react-thesidebar
-- Side menu: https://github.com/Kureev/react-native-side-menu.git
-- Router: https://github.com/t4t5/react-native-router
-
-### Text / Markup
-
-- Markdown: https://github.com/lwansbrough/react-native-markdown
-- HTML: https://github.com/turingou/react-native-html
-
-### Login integrations
-
-- Facebook Login: https://github.com/brentvatne/react-native-login
```

Comment:
At the moment, this isn't a module that can be installed via npm - it's an example project. I should maybe look at bundling this module and the Facebook SDK into a single lib, and when I do that I will add it to react.parts